packages/performance/wafer-map/src/index.ts

```diff
@@ -0,0 +1,54 @@
+import { waferMapTag } from '@ni/nimble-components/dist/esm/wafer-map';
+import type { WaferMapDie } from '@ni/nimble-components/dist/esm/wafer-map/types';
+
+document.addEventListener('DOMContentLoaded', main);
+
+async function waitAnimationFrame(): Promise<void> {
+    return new Promise(resolve => {
+        requestAnimationFrame(() => resolve());
+    });
+}
+
+async function runBenchmark(benchmark: string, dieCount: number): Promise<void> {
+    const waferMapDies = generateDies(dieCount);
+
+    const start = performance.now();
+    const waferMap = document.createElement(waferMapTag);
```

Comment:
Elements should be construucted with `document.createElement` instead of using the constructor function,